requirements.txt

```diff
@@ -1,4 +1,4 @@
-flair>=0.6.1
+flair>=0.6.1,<=0.9
```

Comment:
@dorukkilitcioglu  to confirm my understanding from your reply: If not the upper bound, our unit tests fail with P3.9.

@wddcheng  can you add a comment on this in the changelog please for reference in future.

Reply:
@skadio correct.

Now that I look at changelog, I do feel like it would be good to have a version bump here. That way, we ensure that installing TextWiser thru PyPI also respects the new requirements.

@wddcheng do you have bandwidth to bump the TextWiser version to 1.3.2 (and all the other bits like updating the docs, uploading to PyPI)? If not, I'm happy to address this.

Reply:
sure! i can do that

Reply:
@dorukkilitcioglu just updated the docs

Reply:
Thanks!